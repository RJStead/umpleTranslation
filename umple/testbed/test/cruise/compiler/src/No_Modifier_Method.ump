generate Java "../src-gen-umple";

class NoModifierMethod
{
    void foo() { 
        String s = 1.0;
    }
}
