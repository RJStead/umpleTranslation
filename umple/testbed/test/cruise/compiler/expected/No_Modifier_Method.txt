No_Modifier_Method.ump:6: error: incompatible types: double cannot be converted to String
    String s = 1.0;
               ^
1 error